### ISSUE TITLE
No focus on input control with gluonfx-gradle-plugin' version '1.0.17-SNAPSHOT'

### ISSUE THREAD

**Issue by websystemtechnology** [gluonhq/gluonfx-gradle-plugin#172]
Since a few days using the gluonfx-gradle-plugin' version '1.0.17-SNAPSHOT' all my input controls (both jfoenix, javafx, gluon) do not take focus when clicked and therefore the keyboard does not open for enter text.
I use javafx version: '21-ea+5'

**Comment by jperedadnr** [comment]
Thanks for reporting.
This is Android? or iOS? 
Does 1.0.16 work fine?

**Comment by websystemtechnology** [comment]
At the moment it seems to only occur on Android.
With 1.0.16 everything is ok

**Comment by jperedadnr** [comment]
Can you try gluonfx-gradle-plugin version 1.0.17-SNAPSHOT with javafx.version 19 ?

**Comment by jperedadnr** [comment]
It seems there is a regression starting JavaFX 20-ea+11
Plugin 1.0.16 uses 20-ea+4 (works), 1.0.17-SNAPSHOT is using 21-ea+5 (fails).

Until this gets fixed, with `javafxStaticSdkVersion` you can set any value up until the highest possible for now: 20-ea+9

**Comment by websystemtechnology** [comment]
Thanks, I will continue to use this version and look forward to updates

**Comment by jperedadnr** [comment]
In any case, as a quick workaround you can add this code to your start method:

```
public void start(Stage stage) {
...
        PauseTransition pauseTransition = new PauseTransition(Duration.millis(100));
        pauseTransition.setOnFinished(f -> stage.requestFocus());
        pauseTransition.playFromStart();
}
```

**Comment by websystemtechnology** [comment]
Thanks, I will test this work around in my test environment. Then in production I will continue to use version 1.0.16 for the moment

**Comment by websystemtechnology** [comment]
bug fixed, upgrading to version 1.0.18 everything is ok. Thank you
